pgr-services/src/main/java/org/egov/pgr/util/MigrationUtils.java

```diff
@@ -54,23 +57,25 @@ public MigrationUtils(UserUtils userUtils, PGRConfiguration config, ObjectMapper
          * @return
          */
 
+        ids.removeAll(Collections.singleton(null));
```

Comment:
why is this added?

Reply:
We are passing accountid attribute inside this List of ids data structure as well in which 128 records have the accountid as null. So, I have added this line to remove all the null values as no UUID corresponding to those exists.